subprojects/wrapper/src/crossVersionTest/groovy/org/gradle/integtests/wrapper/WrapperCrossVersionIntegrationTest.groovy

```diff
@@ -60,7 +60,9 @@ class WrapperCrossVersionIntegrationTest extends CrossVersionIntegrationSpec {
         cleanupDaemons(executer, previous)
     }
 
-    @Requires(adhoc = { AvailableJavaHomes.getJdks("1.6", "1.7") })
```

Comment:
This is actually an `or`, not an `and` - so if Java 6 or Java 7 are available.

Reply:
Then they are two different `@Requires` condition